CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+## 0.10.0 (12/29/2016)
+
+### Breaking bug fixes
+
+* Fix oauth2 token refresh (#109) ([@ianmetcalf][])
+
+### Backwards compatible changes
+
+* Remove unused dependencies and upgrade remaining dependencies. (#112) ([@jmdobry][])
```

Comment:
This includes a bump to `request@2.79.0`. As per https://github.com/google/google-auth-library-nodejs/pull/110#issuecomment-266346971, I think we should understand/agree on the implications before releasing this change.